### PR TITLE
Wiki btn and innerHTML new

### PR DESCRIPTION
**Pull request by seangtkelley**

### CODE REVIEW

File: src/ContentPage.jsx [hunk 1]
```diff
@@ -162,7 +163,7 @@ function Section(props) {
   return (
     <div className="container" style={{ paddingBottom: '15px', marginBottom: '15px', borderBottom: "1px solid grey" }}>
       <div className="row">
-        <CustomHeader>{props.section.line}</CustomHeader>
+        <CustomHeader dangerouslySetInnerHTML={{__html: props.section.line}}></CustomHeader>
```
Comment (sjakati98) [inline src/ContentPage.jsx:166]: This is dangerous.

File: src/ContentPage.jsx [hunk 2]
```diff
@@ -63,6 +63,7 @@ const Video = (props) => (
 const AddVideoModal = (props) => (
   <div>
     <div id="buttonOnRight">
+      <a href={`https://en.wikipedia.org/wiki/?curid=${props.wikiID}`} target="_blank" className="btn btn-outline-primary" style={{marginRight: '8px'}}>Go to Wikipedia Page</a>
```
Comment (YuanxunTao) [inline src/ContentPage.jsx:66]: approved.

**Comment by aasnani** [comment]
Tiny diffs 😉

File: src/ContentPage.jsx (hunk 1 above)
Comment (aasnani) [inline src/ContentPage.jsx:166]: I'm not comfortable with living on the edge like you.

Comment (aasnani) [reply src/ContentPage.jsx:166]: MADLAD

File: src/ContentPage.jsx (hunk 2 above)
Comment (sjakati98) [inline src/ContentPage.jsx:66]: HTML is better than Python.

**Review by sjakati98 — APPROVED** [review]
approved.

**Pull request merged**